scripts/json_device_add.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import hashlib
+
+
+def e(variable):
+    return os.environ.get(variable)
+
+
+json_path = "{}{}{}.json".format(e("BIN_DIR"), os.sep, e("IMAGE_PREFIX"))
```

Comment:
Is the prefix used here as suffix? Looks a bit odd.

Reply:
https://github.com/openwrt/openwrt/blob/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5/include/image.mk#L381

I'm not sure what's the $2 making IMAGE_PREFIX to IMAGE_NAME ...

Reply:
The prefix comes before the suffix (.json in this case) - am I understanding something wrong?